weave/engine_trace.py

```diff
@@ -149,15 +149,21 @@ def attributes(self):
             self.span.attributes = {}
         return self.span.attributes
 
-    def set_tag(self, key, val):
+    def set_tag(self, key, unredacted_val, redacted_val=""):
```

Comment:
Is it possible to include `is_pii=False` instead of redacted_val here?
reason being that the users of this function can make a mistake in passing the redacted_value. How to redact a value should be centralized, where as what values are potentially PII can be distributed.

Reply:
ya, the `set_tag` method would just redact it.

Reply:
I see, `"error_count"` metric is the case where `is_pii=True` will not send anything useful. If we chose to go to the is_pii route, I can offer this as a solution:
```
        root_span.set_metric("foo", len(response_payload["node_to_error"]), is_pii=True)
        root_span.set_metric("bar", True, is_pii=True)
```